js/src/personal/amanda/Amanda.page.tsx

```diff
@@ -1,43 +1,28 @@
-import { Button, Center, MantineProvider, Title } from '@mantine/core'
-import { useViewportSize } from '@mantine/hooks'
+import { MantineProvider } from '@mantine/core'
 import { useState } from 'react'
-import styles from './Amanda.module.css'
-import { AmandaMainPage } from './Amanda.main.page.tsx'
+import { MainPage } from './Main.page.tsx'
+import { LandingPage } from './Landing.page.tsx'
 import { theme } from './theme.ts'
 
-export function AmandaPage() {
-  const { height, width } = useViewportSize()
-  const [landingPage, setLandingPage] = useState(1)
+enum Page {
+  Landing,
+  Main,
+}
 
-  return (
-    <MantineProvider theme={theme}>
-      {landingPage === 1 && ( // render loading page
```

Comment:
This style of doing a switch might be cleaner and keeps one instance of MantineProvider for your page.